### ISSUE TITLE
PDF Viewer needs to be removed.

### ISSUE THREAD

**Issue by mikelee-msft** [MicrosoftDocs/OfficeDocs-SharePoint#290]
This section needs to be removed since the PDF viewer will be removed from RTM.

PDF Viewer
SharePoint Server 2019 Public Preview can now render PDF documents on the server. Users no longer need to rely on a PDF viewer client application.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9f0c473c-9953-3f98-5ba5-7fda35879614
* Version Independent ID: 79194e89-ce43-eefc-6439-cf2bd5bdac57
* Content: [New and improved features in SharePoint Server 2019 Public Preview](https://docs.microsoft.com/en-us/sharepoint/what-s-new/new-and-improved-features-in-sharepoint-server-2019)
* Content Source: [SharePoint/SharePointServer/what-s-new/new-and-improved-features-in-SharePoint-Server-2019.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/live/SharePoint/SharePointServer/what-s-new/new-and-improved-features-in-SharePoint-Server-2019.md)
* Product: **sharepoint-server-itpro**
* GitHub Login: @bentoncity
* Microsoft Alias: **stevhord**

**Comment by AndreaBarr** [comment]
@mikelee-msft Thank you for this feedback

It looks like Trevor updated the article and we are just waiting on it to be reviewed and approved for merging. We appreciate the information you provided and the time you are taking to make the docs better.

**Comment by TroyStarr** [comment]
Now that Trevor's change is merged into the public doc, I think we can close out this issue.
